airbyte-workers/src/main/java/io/airbyte/workers/worker_run/TemporalEventRunner.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.worker_run;
+
+import io.airbyte.api.model.ConnectionUpdate;
+import io.airbyte.config.persistence.ConfigNotFoundException;
+import io.airbyte.validation.json.JsonValidationException;
+import io.airbyte.workers.temporal.TemporalClient;
+import io.airbyte.workers.temporal.TemporalClient.ManualSyncSubmissionResult;
+import java.io.IOException;
+import java.util.Set;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class TemporalEventRunner implements EventRunner {
```

Comment:
would it make sense for this to live in `airbyte-scheduler:client` with the other clients over our job infra?

Reply:
Maybe for the interface but should the implementation lives in the worker because of it's dependency on the `TemporalClient`

Reply:
I'm not sure I understand that logic. `TemporalClient` is already used in `airbyte-scheduler:client`. While the name is `airbyte-scheduler` will be wrong and need to be deprecated, it seems like that module contains the same time of code they you're writing here, is shared in the places you need it, and is designed to server the purpose exposing clients for creating jobs which is what you are trying to do.

![Screen Shot 2022-02-28 at 7 17 50 PM](https://user-images.githubusercontent.com/9092207/156098390-116cd556-6a31-4ebd-b982-31fcf6893fd4.png)

Reply:
The main issue I faced when trying to move the TemporalClient is that it has a lot of dependencies on the worker project. It will lead to a circular dependencies.

Reply:
I'm not suggesting moving `TemporalClient`. Just suggesting moving `TemporalEventRunner`. This shouldn't cause any dependency issues.

Reply:
Done.